### ISSUE TITLE
How do I change the typeface / font when reading a news item?

### ISSUE THREAD

**Issue by debajit** [emacs-elfeed/elfeed#190]
The font looks like the default (proportional) sans-serif font (like Helvetica) but I can't figure out how to set it. Can you tell me what this face is called, so that I can customize it?

**Comment by skeeto** [comment]
You could hook elfeed-show-mode (elfeed-show-mode-hook) to set the
face you like.

    (add-hook 'elfeed-show-mode-hook
              (lambda () (buffer-face-set 'variable-pitch)))

That's really all there is to it. However, Emacs cannot correctly wrap
paragraphs with a variable width font, so alignment and wrapping may
look off.

**Comment by mnp** [comment]
Yeah, this was my question too.  People are going to be fussy about reading walls of text and want easy access to appearance.  I read code for a while before coming here. Settings in this region should be easy to find at least: perhaps some defconst's for body and title faces?

**Comment by powellc** [comment]
@mnp thanks so much for following up on your customization efforts. It might be nice to add this sort of code example to the docs so folks don't have to rifle through github issues with font issues to find this.

**Comment by minad** [comment]
Question seems to be answered. Generally Elfeed major mode can be customized via the mode hooks.
